packages/web/src/components/DevFatalErrorPage.tsx

```diff
@@ -7,7 +7,9 @@ import { useState } from 'react'
 import StackTracey from 'stacktracey'
 
 // RWJS_SRC_ROOT is defined and defaulted in webpack to the base path
-const srcRoot = process.env.RWJS_SRC_ROOT || ''
+
+// @MARK error boundary not working in Vite
```

Comment:
This comment broke the connection between the RWJS_SRC_ROOT comment and the usage of RWJS_SRC_ROOT. Is it possible to move the @MARK somewhere else?

Reply:
Yeah I'll remove this, got it working.